de-de/css-de.html.markdown

```diff
@@ -141,7 +141,6 @@ selector {
     font-family: Arial;
     font-family: "Courier New"; /* wenn der Name ein Leerzeichen beinhält, kommt er in
     Anführungszeichen */
-    font-family: "Courier New", Trebuchet, Arial; /* wird die erste Schriftart 
```

Comment:
Sorry GitHub for Desktop for some reason just removed the line. Just added it back in. Sorry for the mistake.